webpack/webpack.config.babel.js

```diff
@@ -1,176 +0,0 @@
-import less from "less";
-import path from "path";
-import postcss from "postcss";
-import purifycss from "purify-css";
-import StringReplacePlugin from "string-replace-webpack-plugin";
-
-function absPath() {
-  const args = [].slice.apply(arguments);
-  args.unshift(__dirname, "..");
-
-  return path.resolve.apply(path.resolve, args);
-}
-
-// Can override this with npm config set externalplugins ../some/relative/path/to/repo
-const externalPluginsDir = absPath(
-  process.env.npm_config_externalplugins || "plugins"
-);
-
-new Promise(function(resolve, reject) {
-  const cssEntryPoint = path.join(__dirname, "../src/styles/index.less");
-  less.render(
-    fs.readFileSync(cssEntryPoint).toString(),
-    {
-      filename: cssEntryPoint,
-      plugins: [colorLighten]
-    },
-    function(error, output) {
-      if (error) {
-        console.log(error);
-        process.exit(1);
-      }
-
-      const prefixer = postcss([autoprefixer]);
-      prefixer
-        .process(output.css)
-        .then(function(prefixed) {
-          resolve(prefixed.css);
-        })
-        .catch(reject);
-    }
-  );
-}).then(function(css) {
-  bootstrap.CSS = css;
-});
-
-function requireFromString(src) {
-  const Module = module.constructor;
-  const sourceModule = new Module();
-  sourceModule._compile(src, "");
-
-  return sourceModule.exports;
-}
-
-const bootstrap = {
-  CSS: ""
-};
-
-module.exports = {
-  lessLoader: {
-    lessPlugins: [colorLighten]
-  },
-
-  module: {
-    preLoaders: [
-      // Replace HTML comments
-      {
-        test: /\.html$/,
-        exclude: /node_modules/,
-        loader: StringReplacePlugin.replace({
-          replacements: [
-            {
-              pattern: /<!--\[if\sBOOTSTRAP-HTML\]><!\[endif\]-->/g,
-              replacement() {
-                return `
-                  <div class="application-preloader">
-                    <div class="application-preloader-spinner"></div>
-                  </div>`;
-              }
-            }
-          ]
-        })
-      },
-      {
-        test: /\.js$/,
-        loader: "source-map-loader",
-        exclude: /node_modules/
-      }
-    ],
-    loaders: [
-      {
-        test: /\.html$/,
-        loader: "html?attrs=link:href"
-      },
-      {
-        test: /\.json$/,
-        loader: "json-loader"
-      },
-      {
-        test: /\.jison$/,
-        loader: "jison-loader"
-      },
-      {
-        test: /\.raml$/,
-        loader: "raml-validator-loader"
-      },
-      {
-        test: /\.(ico|icns)$/,
-        loader: "file?name=./[hash]-[name].[ext]"
-      },
-      {
-        test: /\.(ttf|woff)$/,
-        loader: "file?name=./fonts/source-sans-pro/[name].[ext]"
-      }
-    ],
-    postLoaders: [
-      {
-        test: /\.html$/,
-        exclude: /node_modules/,
-        loader: StringReplacePlugin.replace({
-          replacements: [
-            {
-              pattern: /<!--\[if\sBOOTSTRAP-CSS\]><!\[endif\]-->/g,
```

Comment:
Is this stuff happening somewhere else now, or is one of the webpack plugins taking care of it?

Reply:
we have implemented it here: 👍

Reply:
here: https://github.com/dcos/dcos-ui/pull/2817/files#diff-307854f18e096bcb2404b859d732dcdc